my-contacts/src/Form.js

```diff
@@ -0,0 +1,64 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {useStyles} from "react-treat";
+import * as styleRefs from "./Form.treat";
+
+const Form = (props) =>
+ {
+     const styles = useStyles(styleRefs);
+
```

Comment:
I wonder if it would be better to use one state 

const [contact, setContact] = useState({
image:"",
firstName:"",
etc...
})

that state can updated using a function 

const updateContact = (e) => setContact({...contact} [e.target.value])

Reply:
Really like this!